examples/typescript/exec/exec-example.ts

```diff
@@ -1,9 +1,15 @@
 import k8s = require('@kubernetes/client-node');
 
-let command = process.argv[2];
+const command = process.argv[2];
 
-let kc = new k8s.KubeConfig();
+const kc = new k8s.KubeConfig();
 kc.loadFromDefault();
 
-let exec = new k8s.Exec(kc);
-exec.exec('default', 'nginx-4217019353-9gl4s', 'nginx', command, process.stdout, process.stderr, process.stdin, true /* tty */);
+const exec = new k8s.Exec(kc);
+exec.exec('default', 'demo', 'demo', command,
```

Comment:
I think it would be a nice to be able to show the difference between pod/container here. Either via a comment or via the previous example `'nginx-4217019353-9gl4s` shows more clearly its pod

Reply:
reverted.